jerry-core/ecma/base/ecma-helpers-number.c

```diff
@@ -267,9 +267,25 @@ ecma_number_is_nan (ecma_number_t num) /**< ecma-number */
 ecma_number_t
 ecma_number_make_nan (void)
 {
-  return ecma_number_pack (false,
-                           (1u << ECMA_NUMBER_BIASED_EXP_WIDTH) - 1u,
-                           1u);
+#if ENABLED (JERRY_NUMBER_TYPE_FLOAT64)
+  union
+  {
+    double dbl;
+    uint64_t ui;
+  } f;
+  f.ui = 0x7ff8000000000000;
```

Comment:
I would add an ull postfix after the number, or make it from two uint32 value.

Reply:
Done.